AdminWebsite/AdminWebsite/Controllers/JusticeUsersController.cs

```diff
@@ -58,5 +59,42 @@ public async Task<ActionResult> AddNewJusticeUser([FromBody] AddJusticeUserReque
                 throw;
             }
         }
+
+        /// <summary>
+        /// Delete a justice user
+        /// </summary>
+        /// <param name="id">The justice user id</param>
+        /// <returns></returns>
+        [HttpDelete("{id}")]
+        [SwaggerOperation(OperationId = "DeleteJusticeUser")]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.NoContent)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotFound)]
+        [ProducesResponseType(typeof(ValidationProblemDetails), (int)HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> DeleteJusticeUser(Guid id)
+        {
+            try
+            {
+                await _bookingsApiClient.DeleteJusticeUserAsync(id);
+
+                return NoContent();
+            }
+            catch (BookingsApiException e)
+            {
+                if (e.StatusCode is (int)HttpStatusCode.NotFound)
+                {
+                    var typedException = e as BookingsApiException<string>;
+                    return NotFound(typedException.Result);
+                }
+
+                if (e.StatusCode is (int)HttpStatusCode.BadRequest)
+                {
+                    var typedException = e as BookingsApiException<ValidationProblemDetails>;
```

Comment:
Because you're using _as_ to cast, is there an chance the 400 from the booking-api will not match the target 'ValidationProblemDetails', if its a bad request and it doesnt, we'll null reference exception when calling typedException.Result on the next line.